### PR TITLE
Failure Fix

### PR DESCRIPTION
**Pull request by SamuelWM01**
Attempted fix to insecure temp file caused failures.
Changing it back for the time being.

### CODE REVIEW

**Comment by esproul** [comment]
```
utils/mtev_stacktrace.c:122: error: 'oldmask' undeclared (first use in this function)
```

Failures only happened on Linux, and not on OmniOS, but according to Sam, simply declaring the `oldmask` variable beforehand did not work.

**Comment by SamuelWM01** [comment]
When I had the variable declared before hand it crashed and burned in my devzone, looking into it rn.

File: src/utils/mtev_stacktrace.c [hunk 1]
```diff
@@ -119,9 +119,7 @@ void mtev_stacktrace(mtev_log_stream_t ls) {
     /* This is Async-Signal-Safe (at least on Illumos) */
     char tmpfilename[MAXPATHLEN];
     snprintf(tmpfilename, sizeof(tmpfilename), "/var/tmp/mtev_%d_XXXXXX", (int)getpid());
-    oldmask = umask(0600);
```
Comment (postwait) [inline src/utils/mtev_stacktrace.c]: Declare it here... 

```
int oldmask = umask(0600);
```

Comment (SamuelWM01) [inline src/utils/mtev_stacktrace.c]: just did that, It seems to be working in my devzone so I'll push it back up.

**Pull request merged**